test/matlab_experimental/ctTestSetUp.m

```diff
@@ -0,0 +1,20 @@
+clear all
+
+% Copy library to test folder
+ctTestPath;
+
+% Load Cantera
+rootDir = getenv('CANTERA_ROOT');
+ctName = '/test/matlab_experimental/libcantera_shared.so';
+% Load Cantera
+if ~libisloaded('libcantera_shared')
+    [~, warnings] = loadlibrary([rootDir, ctName], ...
+                                [rootDir, '/include/cantera/clib/ctmatlab.h'], ...
+                                'includepath', [rootDir, '/include'], ...
+                                'addheader', 'ct', 'addheader', 'ctfunc', ...
+                                'addheader', 'ctmultiphase', 'addheader', ...
+                                'ctonedim', 'addheader', 'ctreactor', ...
+                                'addheader', 'ctrpath', 'addheader', 'ctsurf');
+end
```

Comment:
I did not use `ctLoad` since it reads the value of `ctRoot` which (for now) is set by the user. For the Github actions, the test set up uses environment variable `CANTERA_ROOT` instead.

Reply:
I think there should be a way to avoid needing to do this, but we can certainly work on that as a separate improvement.